general_considerations.md

```diff
@@ -313,6 +447,41 @@ and immediately paid out to hashers, as there are not enough known parties to
 sign a multi-signature, and we don't even know their pubkeys to construct a
 (P2TR, P2SH, etc) bitcoin output address and scriptPubKey.
 
+[kp]
+
+Can we roll over the payout of the first four blocks to be evenly split between
+miners mining the first five blocks.  What you proposed will work in a new
+network, but in bitcoin mining directly paying the first four miners without
+sharing the rewards will be a deal breaker for any miner.
+
+Another point is that bootstrapping this pool will be really impossible. We are
+asking four large miners to get together to bootstrap the pool. This won't be
+practical.
+
+Instead, we should think of a way that members can join the pool, start
+contributing shares and once anyone gets lucky, the entire pool gets the
+reward. Intuitively, this seems like a more practically feasible way to
+bootstrap the pool.
+
+If change the requirement that signers have already found a block to be
+something along the lines that signers collectievly include 75% of the hashrate,
+then we can still require that enough hashrate is committing to a payout and at
+the same can bootstrap the network.
+
+It could work something like this:
+
+1. All miners have the same DAG view, so if they are not malicious they will
+   construct the same UHPO view.
+2. They run a DKG/TSS round to sign the payout with top 66%+1 of the hashrate
+   providers included in the TSS signing group.
+3. A BFT agreement is reached to sign the payout in the form of the threshold
+   signature.
+4. We will need to make sure all participants run a determinisitic algorithm to
+   construct the UHPO view. We can provide for this with simple tie-breaking
+   rules.
+
+[/kp]
+
```

Comment:
I don't think we have any disagreement here. "Tie breaking" in #4 isn't required because the payout happens at the 5th block paying the shares from the DAG from genesis to the 4th block. (Assuming all that is more than 100 bitcoin blocks back) The DAG consensus itself is deterministic.

The rules about who signs the payout for the first few blocks is encoded in the node software, we don't have to ask large miners to do anything except run it (and it's entirely possible one of the set will be a small miner).

I'm confused about what's confusing here...